pkg/cli/x_ips.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2015 Scaleway. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE.md file.
+
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/scaleway/scaleway-cli/pkg/api"
+)
+
+var cmdIPS = &Command{
+	Exec:      runIPS,
+	UsageLine: "_ips [OPTIONS] [ARGS]",
+
+	Description: "",
+	Hidden:      true,
+	Help:        "Display your ips",
+	Examples: `
+    $ scw _ips
+    $ scw _ips IP_ID
+    $ scw _ips --new
+    $ scw _ips --update IP_ID:SERVER_ID
```

Comment:
Yes, that's why I'm not 100% sure it's fully wrong

I think colon is useful in two cases:

* mapping, as docker do for volumes: `docker run -v /local/path:/container/path image`
* complex identifier but still only one object, i.e: "server:123456"

---

In this particular case, I think --update IP_ID:SERVER_ID is not clear at all and I would move the function to `scw _patch IP_ID server=SERVER_ID` or `scw _ips --attach IP_ID SERVER_ID` without colon

About the security group, I think the case is too complex for now, I suggest to wait a little and refactor later